examples/fundamentals__fixtures/cypress/integration/load-fixtures-spec.js

```diff
@@ -0,0 +1,114 @@
+/// <reference types="cypress" />
+describe('Loading multiple fixtures', () => {
+  context('before each test using closure variables', () => {
+    let city
+    let country
+
+    beforeEach(() => {
+      cy.fixture('city').then(c => {
+        city = c
+      })
+      cy.fixture('country').then(c => {
+        country = c
+      })
+    })
+
+    it('has loaded fixtures', () => {
+      expect({ city, country }).to.deep.equal({
+        city: { name: 'Atlanta' },
+        country: { name: 'United States' }
+      })
+    })
+  })
+
+  context('once before all tests', () => {
+    let city
+    let country
+
+    before(() => {
+      // load fixtures once before any tests
+      // and they are kept in closure variables
+      cy.fixture('city').then(c => {
+        city = c
+      })
+      cy.fixture('country').then(c => {
+        country = c
+      })
+    })
+
+    it('has loaded fixtures', () => {
+      expect({ city, country }).to.deep.equal({
+        city: { name: 'Atlanta' },
+        country: { name: 'United States' }
+      })
+    })
```

Comment:
are you positive this works when you add another test?

Reply:
yup, and I have added second tests to confirm this